packages/components/src/components/navigation/ProductMenuSection.tsx

```diff
@@ -156,7 +160,7 @@ export class ProductMenuSection extends React.Component<MenuSectionProps, any> {
             const seeAllUrl = config.seeAllURL || AppURL.create(section.key);
             columns.push(
                 <span className="overflow-link" key="overflow">
-                    <a href={seeAllUrl.toHref()}>See all {section.totalCount}</a>
+                    <a href={typeof seeAllUrl == 'string' ? seeAllUrl : seeAllUrl.toHref()}>See all {section.totalCount}</a>
```

Comment:
This "typeof url is string" check happens a few times in this PR, any reason to move it to a little util helper?

Reply:
Good idea.  Done.